library/core/src/error.rs

```diff
@@ -219,13 +214,18 @@ impl<'a> dyn Error + 'a {
     /// Request a reference of type `T` as context about this error.
     #[unstable(feature = "error_generic_member_access", issue = "99301")]
     pub fn request_ref<T: ?Sized + 'static>(&'a self) -> Option<&'a T> {
-        core::any::request_ref(self)
+        request_ref(self)
     }
 
     /// Request a value of type `T` as context about this error.
     #[unstable(feature = "error_generic_member_access", issue = "99301")]
     pub fn request_value<T: 'static>(&'a self) -> Option<T> {
-        core::any::request_value(self)
+        request_value(self)
+    }
+
+    /// Request a ref of type `T` from the given `Self` instance.
+    pub fn request_ref_from<T: ?Sized + 'static>(this: &'a Self) -> Option<&'a T> {
+        this.request_ref()
```

Comment:
What is the point of this method?

Reply:
Honestly, it's been about a month so I don't remember. My first impulse is to say "it's a naive copy pasta" but I only see it being added in the diff, not removed elsewhere. Maybe I copied a code example from an RFC and wanted to make doctests compile/pass... :thinking:

Reply:
Oh yeah, here it is: https://github.com/rust-lang/rfcs/pull/3461/files#diff-bb91f37510dc7a6369183c40dbdcfb52164335efa7a04f005a906d0006754d27R419

Reply:
No wait, I'm the one who added that to the RFC: https://github.com/rust-lang/rfcs/pull/3461/commits/0cffa52cca1a4b12d8364b15fb67f561a9e8b8a7#diff-bb91f37510dc7a6369183c40dbdcfb52164335efa7a04f005a906d0006754d27R419

...what in the devil?

Reply:
I'll remove it :)